ref/general/requestTiming.adoc

```diff
@@ -22,6 +22,12 @@ For a hung request, apart from printing a warning message along with the relevan
 Three Java cores are generated for a hung request, each a minute apart.
 If a previously hung request is completed, a detailed message is logged, including the ID and time of completion of the request.
 
+A slow request can be slow because the code path length of a request is particularly long or because some resource the request depends on, such as a database is busy.
+Requests that take much longer, and which are therefore likely to be hung, are typically blocked by lack of access to a shared resource, for example, a section of code that only allows serial access and transfers information sequentially.
+In the case of a hung request, this can be blocked for a long time or even indefinitely.
```

Comment:
Avoid using "this" as subject of clause: 

`A hung request can be blocked for a long time or even indefinitely.`